tests/transport_test.go

```diff
@@ -60,12 +60,21 @@ var _ = Describe("Transport Tests", func() {
 			sec *v1.Secret
 		)
 
+		var stringData map[string]string
 		if credentialed {
 			By(fmt.Sprintf("Creating secret for endpoint %s", ep))
-			stringData := map[string]string{
-				common.KEY_ACCESS: utils.AccessKeyValue,
-				common.KEY_SECRET: utils.SecretKeyValue,
+			if accessKey != "" && secretKey != "" {
+				stringData = map[string]string{
+					common.KEY_ACCESS: accessKey,
+					common.KEY_SECRET: secretKey,
+				}
+			} else {
+				stringData = map[string]string{
+					common.KEY_ACCESS: utils.AccessKeyValue,
```

Comment:
If the credentials aren't supplied, they should be left empty

Reply:
changed the entire logic as above